tests/README.md

```diff
@@ -78,45 +78,45 @@ $ sudo snap install ./checkbox-tdx-classic_2.0_amd64.snap --dangerous --classic
 - Run sanity tests to check the host setup:
 
 ```
-$ sudo -E checkbox-tdx-classic.test-runner-automated-host
+$ checkbox-tdx-classic.test-runner-automated-host
```

Comment:
I saw your comment about `sudo` only being needed because reports were previously in `/tmp`. I'm curious why we need `sudo` for the tox tests but not with checkbox?

Reply:
Another reason to need sudo is the need in "tests/test_host_tdx_hardware.py" to access msr register.

Reply:
Also some tests require rmmod kvm_intel and insmode

Reply:
Also cpu pinning requires sudo

Reply:
@frenchwr @spmcmillan there are 2 modes of running checkbox tests : direct or remote. with checkbox provider, we usually have a checkbox agent deamon (systemd service for tdx : `snap.checkbox-tdx-classic.remote-slave.service`), this deamon (command: `python3 /snap/checkbox22/current/bin/checkbox-cli slave`) runs as root and is the runner of the tests. per consequence, the tests run under root. When we issue `checkbox-tdx-classic.test-runner-automated-host`, we actually request (via RPC by issuing `checkbox-cli remote 127.0.0.1 ...`) the daemon to run the tests on our behalf.

Reply:
makes sense